Dockerfile

```diff
@@ -1,11 +1,17 @@
 ARG coq_image="coqorg/coq:dev"
 FROM ${coq_image} as builder
 
+ENV MATHCOMP_VERSION="dev"
+ENV MATHCOMP_PACKAGE="coq-mathcomp-character"
+
 WORKDIR /home/coq/mathcomp
 
 COPY . .
 
-RUN ["/bin/bash", "--login", "-c", "set -x \
+SHELL ["/bin/bash", "--login", "-o", "pipefail", "-c"]
+
+# hadolint ignore=SC2046
```

Comment:
?

Reply:
To give more insight, `RUN` admits two syntaxes:

* the **shell form**:  
  `RUN cmd1 && cmd2`  
  (which uses `/bin/sh -c` as implicit shell, or `/bin/bash --login … -c` thanks to the previous `SHELL` command)

* the **exec form**:  
  `RUN ["prog", "arg1", "…"]`  
  which is more "precise" because we can specify the shell in an explicit way, but is very rarely used in practice because it's heavier.

So this patch is just a convenience refactoring that amounts to remove one level of brackets/quoting.

Regarding # hadolint ignore, it's just a Dockerfile linter annotation I had added when preparing Dockerfile for stable versions of math-comp.

I've just run it again (`docker run --rm -i hadolint/hadolint < Dockerfile`) and it spotted more details; will force-push another commit.